xml/cl.xml

```diff
@@ -3730,6 +3737,12 @@ server's OpenCL/api-docs repository.
             <param><type>cl_uint</type>                                 <name>arg_index</name></param>
             <param>const <type>void</type>*                             <name>arg_value</name></param>
         </command>
+        <command suffix="CL_API_SUFFIX__VERSION_3_0">
+            <proto><type>cl_int</type>                                  <name>clSetKernelArgDevicePointerEXT</name></proto>
+            <param><type>cl_kernel</type>                               <name>kernel</name></param>
+            <param><type>cl_uint</type>                                 <name>arg_index</name></param>
+            <param>const <type>cl_mem_device_address_ext</type>*        <name>arg_value</name></param>
```

Comment:
I don't think this is correct, and this doesn't match the function signature in POCL.  I think it should be:

```suggestion
            <param><type>cl_mem_device_address_ext</type>               <name>arg_value</name></param>
```

It'd probably be a good idea to generate the headers from this XML file to make sure everything looks as expected.

Reply:
Hmm, the signature in the XML file *does* seem to match the function signature in Mesa though.

@karolherbst and @pjaaskel, which one is right?

Reply:
I believe what you @bashbaug propose is right. I don't see point passing a pointer to the pointer value instead of passing it by value. @karolherbst do you agree?

Reply:
I think we had a discussion on this and potentially considered passing a list of addresses one per device. And I assumed this change was made, so it's easier to switch to that model when decided so in a future version of this extension.

Though I don't mind either way.

Reply:
Reverted to value passing.